client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -239,7 +241,7 @@ abstract class BaseActivity(
      * Get value from language resource using the resource Id and language Id
      * By value we mean what a particular word is known in a particular language
      */
-    protected suspend fun getValueFromName(@StringRes resId: Int, languageId: Int = appLanguageId!!): String {
+    suspend fun getValueFromName(@StringRes resId: Int, languageId: Int = appLanguageId!!): String {
```

Comment:
Why are we removing the visibility modifier here?

Reply:
We were using it from the fragment. Now since we are not retrieving strings from the database we no longer need this method